nestkernel/event.h

```diff
@@ -45,41 +45,59 @@ namespace nest
 class Node;
 
 /**
- * Encapsulates information which is sent between Nodes.
+ * Encapsulate information sent between Nodes.
```

Comment:
```suggestion
 * Encapsulate information sent between nodes.
```

Reply:
I think nodes should not be capitalized, even if the class is. It seems to me most text we have regarding nodes is lower case.